app.py

```diff
@@ -33,7 +33,7 @@
 app.register_blueprint(app_predictions)
 app.register_blueprint(discord)
 
-if not app.debug or os.environ.get('WERKZEUG_RUN_MAIN') == 'true':
```

Comment:
this line stops flask from initialising twice, we still need it
`if not app.debug or os.environ.get('WERKZEUG_RUN_MAIN') == 'true':`
https://stackoverflow.com/questions/9449101/how-to-stop-flask-from-initialising-twice-in-debug-mode

Reply:
I've added those back in.